include/readout/RawWIBTp.hpp

```diff
@@ -0,0 +1,235 @@
+/**
+ * @file RawWIBTp.hpp Raw Trigger Primitive bit fields and accessors
+ *
+ * This is part of the DUNE DAQ , copyright 2020.
+ * Licensing/copyright details are in the COPYING file that you should have
+ * received with this code.
+ */
+#ifndef DATAFORMATS_INCLUDE_DATAFORMATS_WIB_RAWWIBTP_HPP_
+#define DATAFORMATS_INCLUDE_DATAFORMATS_WIB_RAWWIBTP_HPP_
+
+#include <bitset>
+#include <iostream>
+#include <vector>
+
+namespace dunedaq {
+namespace dataformats {
+
+typedef uint32_t word_t;
```

Comment:
The `WIBFrame` header also defines `using word_t = uint32_t;` in the `dunedaq::dataformats` namespace. I'm not sure if this could ever end up being a problem (maybe if you included both headers?), but maybe it's safer to change the typedef name (`tp_word_t`?) or put it inside one of the classes?

Reply:
I would choose to change it to `tp_word_t`. That's a good point, thanks.

Reply:
Anyway we need to move this class to `dataformats` for v2.4. So I would propose to follow up on these after our discussion in two days.